lib/stbcbckt.gi

```diff
@@ -1340,10 +1333,6 @@ InstallGlobalFunction( PartitionBacktrack,
                     image.level := rbase.lev[ d ];
                     if IsSlicedPerm( image.perm )  then
                         image.perm!.length := oldprm;
-                         # Here and below the code that refers to `rgtObj` was used to avoid multiplication
```

Comment:
I think this is actually a useful comment (in particular if ever there will be a discrepancy to an older version.) I would leave this in.

Reply:
This comment was added by myself. @AnnaKDS and I had a look at the backtrack code in 2018 for a seminar and @ChrisJefferson and @markuspf helped us to understand it. There we noticed that this was still documented but not used in the code. So I removed it from the documentation but added this comment so that its easier to understand why the code is the way it is. The relevant PR is #3101